### PR TITLE
Froude number

### PR DESCRIPTION
**Pull request by aokiyoshi**
added law and tests

### CODE REVIEW

File: symplyphysics/laws/hydro/froude_number.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+# Description
+# Froude number characterizes the ratio between the force of inertia and
+# the external force, in the field of which the motion occurs, acting on
+# an elementary volume of liquid or gas.
+# Law: Fr = u / sqrt(g * L), where
+# u is velocity,
+# g is acceleration due to gravity,
+# L is a characteristic length.
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/froude_number.py:13]: What's this?

File: symplyphysics/laws/relativistic/relativistic_time_dilation.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from sympy import Eq, solve, sqrt
```
Comment (blackyblack) [inline symplyphysics/laws/relativistic/relativistic_time_dilation.py:1]: Remove unrelated law.

Comment (aokiyoshi) [reply symplyphysics/laws/relativistic/relativistic_time_dilation.py:1]: That is from other law. I forgot to create another branch for it

File: symplyphysics/laws/hydro/froude_number.py (hunk 1 above)
Comment (blackyblack) [reply symplyphysics/laws/hydro/froude_number.py]: Not addressed. What's this length?

Comment (aokiyoshi) [reply symplyphysics/laws/hydro/froude_number.py]: What should I do? Provide further description for L?

Comment (blackyblack) [reply symplyphysics/laws/hydro/froude_number.py]: Yes. What is it?

File: symplyphysics/laws/hydro/froude_number.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+# Description
+# Froude number characterizes the ratio between the force of inertia and
+# the external force, in the field of which the motion occurs, acting on
+# an elementary volume of liquid or gas.
+# Law: Fr = u / sqrt(g * L), where
+# u is velocity,
+# g is acceleration due to gravity,
+# L is a characteristic length (the characteristic length is the dimension
+# that defines the length scale of a physical system),
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/froude_number.py:14]: Sorry but it is very hard to understand what it is. I see that you took it from Wikipedia, but could you explain it in layman terms? You can also add a waterline of a ship, as an example of characteristic length.

File: symplyphysics/laws/hydro/froude_number.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+# Law: Fr = u / sqrt(g * L), where
+# u is velocity,
+# g is acceleration due to gravity,
+# L is a characteristic length (The characteristic length is the dimension
+# that defines the length scale of a physical system. A characteristic length
+# is usually the volume of a system divided by its surface: L = V / A,
+# where V is the volume of the body, and A is the cross-sectional area.
+# For example, it is used to calculate flow through circular and non-circular
+# tubes in order to examine flow conditions. D = 4 * A / p, where
+# D is characteristic diameter, A is the cross-sectional are, p is wetted perimeter),
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/froude_number.py:19]: Still not perfect, but let's keep it this way. Please move it to description, to avoid cluttering the law.

**Review by blackyblack — APPROVED** [review]

**Review by blackyblack — APPROVED** [review]
